### ISSUE TITLE
Match unit search with main site

### ISSUE THREAD

**Issue by mhk33** [cul-it/uls#1217]
Pending https://github.com/cul-it/wp-cul-main/issues/540

Lenora asked if the units could have the same search options as the main site (catalog, articles, etc.).

**Comment by sarah-cul** [comment]
@JesiBuell79 @mhk33 

With the recent changes to the search on the main library site, this request may no longer be applicable?  The option to choose the resource is no longer there on the main site.

When doing a search on a unit site and choosing the default "Library Resources" it does the same search and gives the same results as the main site when choosing "All Resources". 

What do you think?

**Comment by mhk33** [comment]
@sarah-cul Agree, I don't think this is applicable any more and it would be okay to close it.
